lib/invoca/metrics/prometheus/declare_metrics/counter.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require_relative 'base'
+
+module Invoca
+  module Metrics
+    module Prometheus
+      module DeclareMetrics
+        class Counter < Base
+
+          def increment(value = 1, **labels)
+            metric.present? and metric.increment(default_labels.merge(labels), value)
```

Comment:
I think this is better flipped to read as the following

```suggestion
            metric.increment(default_labels.merge(labels), value) if metric.present?
```

Reply:
Added in e48cdb60a41d3d97c5d5521f19c533cec521cd6b